src/main/java/core/basesyntax/services/ReadFromFileService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.services;
+
+import java.util.List;
+
+public interface ReadFromFileService {
+    List<String> readDbFromFile(String fileName);
```

Comment:
```suggestion
    List<String> readFile(String fileName);
```

Ми тит не зчитуємо ніяку базу данних, просто зчитуємо файл